set.mm

```diff
@@ -180023,6 +180111,30 @@ obviously not a bijection (by Cantor's theorem ~ canth2 ), and in fact
     ( cz wcel c1 cgcd co wceq 1z gcdcom mpan gcd1 eqtrd ) ABCZDAEFZADEFZDDBCMNO
     GHDAIJAKL $.
 
+  ${
+    gcdmuld.1 $e |- ( ph -> M e. NN0 ) $.
+    gcdmuld.2 $e |- ( ph -> N e. ZZ ) $.
+    $( The greatest common divisor of a nonnegative integer and a multiple of
+       it is itself.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    gcdmuld $p |- ( ph -> ( M gcd ( N x. M ) ) = M ) $=
```

Comment:
This is a generalzation of ~gcdmultiple and ~gcdmultiplez. Since it is already placed before ~gcdmultiple, it should be renamed ~gcdmultipled, and the proofs of ~gcdmultiple and ~gcdmultiplez can be shortened accordingly.

Reply:
Comment should be "The greatest common divisor of a nonnegative integer and a multiple of it is the nonnegative integer itself."

Reply:
Renamed to ` gcdmultipled `. In the comment I referenced `M` to avoid repeating "nonnegative integer" twice.

Reply:
What about using ~gcdmultipled to shorten the proofs of ~gcdmultiple and ~gcdmultiplez? I have pout it oin my TODO list and will look at it later,

Reply:
I haven't been able to shorten those with the automatic minimize tool, so perhaps that requires some manual labor.